pkg/kubectl/drain/examples.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package drain
+
+import (
+	"fmt"
+	"os"
+
+	corev1 "k8s.io/api/core/v1"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+)
+
+// This file contains examples on how to drain/cordon/uncordon nodes.
+// These functions may be called directly, or their functionality
+// copied into your own code, for example if you want different output
+// behaviour.
+
+// RunNodeDrain shows the canonical way to drain a node.
+// You should first cordon the node, e.g. using RunCordonOrUncordon
+func RunNodeDrain(drainer *Helper, nodeName string) error {
```

Comment:
This function along with RunCordonOrUncordon does pretty much everything the end user would want.  We should just put these in drain.go rather than put them in 'examples' to give them more prominence.

Reply:
Good call.  I think it's nice to still have the separation between "plumbing vs porcelian", so I called it "default.go" - does that work?

Reply:
sounds good to me.